typescript/codepipeline-build-deploy/app/buildspec.yaml

```diff
@@ -22,6 +23,8 @@ phases:
       - sed -i "s|EXECUTION_ROLE_ARN|${EXECUTION_ROLE_ARN}|g" taskdef.json
       - sed -i "s|TASK_DEFINITION_ARN|${TASK_DEFINITION_ARN}|g" appspec.yaml
       - cat appspec.yaml && cat taskdef.json
+      - mv appspec.yaml ../
```

Comment:
Should be copy, not move, or the file should be there to begin with. 

I'd suggest outputting to a new file rather than modifying the file in-situ with sed.

Reply:
The sed part is not part of this PR. Though if we'd like to address, would make for a good new issue.

Regarding the move. I tried to reference the file not from root and it didn't work. I can take another swing at that one, though.